smoke-test.sh

```diff
@@ -168,7 +174,8 @@ function doCommand() {
             binlogHttpsPart="https"
         fi
 
-        binlog="$testingDir/${lang}_${proj}_${binlogOnlinePart}_${binlogHttpsPart}_$1.binlog"
+        # double-hyphen and space in the binlog path screw up MSBuild argument parsing
+        binlog=$(echo "$testingDir/${lang}_${proj}_${binlogOnlinePart}_${binlogHttpsPart}_$1.binlog" | sed 's/--/_/g ; s/ /_/g')
```

Comment:
Are there any cases where this is needed?

Reply:
No, shouldn't be any more.  Removed.